packages/useful-types/README.md

```diff
@@ -109,18 +109,18 @@ The following type aliases are provided by this library:
 
 - `DeepOmit<T, K>` Recursively maps over all properties in a type and omits those matching `K`.
 
-```ts
-interface Obj {
-  __typename: string;
-  foo: string;
-  bar: {
+  ```ts
```

Comment:
I've also indented this block as it seems broken: https://github.com/Shopify/quilt/tree/main/packages/useful-types